src/main/resources/application.yaml

```diff
@@ -110,10 +133,33 @@ quarkus:
   # adapt path of the open api definitions
   smallrye-openapi:
     path: /api/json/openapi
+    info-title: ${quarkus.application.name}
+    info-version: ${quarkus.application.version:}
+    info-description:
+    info-terms-of-service:
+    info-contact-name: Stargate
```

Comment:
Should this be not the stargate details instead say and point to data-api documentation

Reply:
Certainly, I could improve these.
Similarly to other documentations, currently just focus on making sure no functionality mistake.